packages/js/onboarding/src/utils/countries/index.ts

```diff
@@ -51,7 +51,14 @@ export const findCountryOption = (
 			countryCode === location.country_short &&
 			option.label.includes( '—' )
 		) {
-			const wcRegion = option.label.split( '—' )[ 1 ].trim();
+			// WP GEO API Returns regions without accents.
+			// Remove accents from the region to compare.
+			// Málaga -> Malaga
+			const wcRegion = option.label
+				.split( '—' )[ 1 ]
+				.trim()
+				.normalize( 'NFKD' )
```

Comment:
I think [localeCompare()](https://dev.to/mliakos/use-localecompare-to-ignore-string-case-and-diacritics-accents-1gbf) might be the most recent way to do this, with locale either undefined or 'en', and { sensitivity: base } ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#parameters))

<img width="782" alt="image" src="https://github.com/woocommerce/woocommerce/assets/27843274/e47408af-b763-40dc-b1b5-75a2ed8ee63c">

Reply:
Thank you for the suggestion! TIL 👍 I'll update it soon.

Reply:
@rjchow updated in https://github.com/woocommerce/woocommerce/pull/39110/commits/fbd12eccdf831e20cba4fdc6bf71970fae18553c

Reply:
thanks! looks good to me!